py/server/deephaven/time.py

```diff
@@ -155,1885 +135,485 @@ def time_zone_alias_rm(alias: str) -> bool:
 
 # endregion
 
-# region Conversions: Time Units
-
+#TODO: Review all function names: to_j_<xyz>?  to_db_<xyz>?  to_dh_<xyz>?  to_<xyz>? as_j_<xyz>?  as_db_<xyz>?  as_dh_<xyz>?
+#TODO: Should the doc strings refer to "Deephaven" or "Java" types?
```

Comment:
I would think we want the docstrings to refer to Deephaven types. I think a user either understands that Deephaven types are implicitly Java types under the hood and doesn't need the docstring to remind them, or they don't care at all and will do just fine being shielded from all the Java stuff. This assumes that the API does not drag in anything that doesn't feel like a Deephaven type.